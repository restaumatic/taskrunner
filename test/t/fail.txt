echo "I will fail" >&2
exit 42
