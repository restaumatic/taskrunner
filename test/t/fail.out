-- output:
[sh] stderr | I will fail
-- exit code: 42
